src/common/offline_transformations/include/compress_quantize_weights.hpp

```diff
@@ -58,7 +61,46 @@ class CompressQuantizeWeights;
     With that we can skip same calculations in the runtime and make loading of such sub-graphs to the plugin faster.
     Additionally zero point can be fused to weights if it doesn't affect accuracy.
 */
-class ov::pass::CompressQuantizeWeights : public ov::pass::MatcherPass {
+class ov::pass::CompressWeightsWithFakeQuantize : public ov::pass::MatcherPass {
+public:
+    OPENVINO_RTTI("CompressWeightsWithFakeQuantize", "0");
+
+    CompressWeightsWithFakeQuantize();
+};
+
+/*
+    CompressWeightsWithFakeConvert replaces FakeConvert node with constant inputs to the following subgraph:
+
+            +----------+
```

Comment:
Can we skip adding Subtract if all the values in shift vector are zero? as it is for FakreQuantize

Reply:
Yes, please take a look on https://github.com/openvinotoolkit/openvino/pull/22433/files#diff-3093fd5e5a72e49a3f351d297d732fa666546d51b374beea94fe438941ad7b45R297